scripts/sort-dependencies

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
```

Comment:
Nice one @kittaakos ❤️ 
What if we run this script in a pre-commit with `husky`?

Also, I would probably rename it with `.js` extension. Or is there a reason why you did otherwise?

Reply:
> What if we run this script in a pre-commit with `husky`?

Good idea. Feel free to enhance it once it's on the main branch.

> Also, I would probably rename it with `.js` extension. Or is there a reason why you did otherwise?

Thanks! As mentioned in the [usage](https://github.com/arduino/arduino-ide/pull/1662/files#diff-5c94e70dafc7f45eebe0530f45ba5abaedfdbc5fe87a4fe0e3fd1133ed99433eR4), I prefer to run it as an executable from a shell. This is how the `bin` hoisting works with yarn on POSIX. Check the files under `./node_modules/.bin` as an example. Of course, it will fail from CMD.exe, but you can run it as `node scripts\\sort-dependencies package.json`. Or if you have a shell on Windows or using GitBash, `./scripts/sort-dependencies ./arduino-ide-extension/package.json` works. Why do you think we should have the `.js` extension?

Reply:
> Why do you think we should have the .js extension?

I suggested it to be consistent with other scripts we already have in the `scripts` folder, but I see your point. Maybe it would make sense to change the others into shell executable scripts (I'm not suggesting to do it in this PR).